github-issues.go

```diff
@@ -129,10 +125,6 @@ func populateIssues(url string) {
 		}
 		if flag != 1 {
 			gIssues = append(gIssues, eachGitIssue)
-			flag = 0
-		} else {
-			// this is not an open issue so don't do anything but continue.
-			flag = 0
```

Comment:
I believe `var flag int` declaration and automatic initialization at the top of the for loop is good enough to set the `flag` variable to `0`. (However, it needs to be moved inside the for loop)
I guess it would be clearer if we discuss it by going through the code.